lib/menubar/init.lua

```diff
@@ -39,6 +39,33 @@ local function get_screen(s)
     return s and capi.screen[s]
 end
 
+
+--- Menubar normal text color.
+-- @beautiful beautiful.menubar_fg_normal
+local fg_color = theme.menubar_fg_normal or theme.menu_fg_normal or theme.fg_normal
+
+--- Menubar normal background color.
+-- @beautiful beautiful.menubar_bg_normal
+local bg_color = theme.menubar_bg_normal or theme.menu_bg_normal or theme.bg_normal
+
+--- Menubar border width.
+-- @beautiful beautiful.menubar_border_width
+-- @tparam[opt=0] number menubar_border_width
+local border_width = theme.menubar_border_width or theme.menu_border_width or 0
+
+--- Menubar border color.
+-- @beautiful beautiful.menubar_border_color
+local border_color = theme.menubar_border_color or theme.menu_border_color
+
+--- Menubar selected item text color.
+-- @beautiful beautiful.menubar_fg_normal
+local focused_fg_color = theme.menubar_fg_focus or theme.menu_fg_focus or theme.fg_focus
+
+--- Menubar selected item background color.
+-- @beautiful beautiful.menubar_bg_normal
+local focused_bg_color = theme.menubar_bg_focus or theme.menu_bg_focus or theme.bg_focus
+
```

Comment:
I guess this should be done within the function. I there is at least 1 person who try to implement changing from dark theme to light theme at runtime without restarting Awesome.

And yes, there is a lot of broken things related to dynamic theme changes, but lets try not to add more.

Reply:
done

Reply:
thanks

Reply:
your last review got dismissed